code/obj/item/card.dm

```diff
@@ -163,6 +163,9 @@ TYPEINFO(/obj/item/card/emag)
 		access = get_access("Captain")
 		src.AddComponent(/datum/component/log_item_pickup, "Captain")
 
+/obj/item/card/id/nt
+	icon_state = "id_nt"
+
```

Comment:
```suggestion
/obj/item/card/id/nt
	icon_state = "id_nt"
	keep_icon = TRUE
	
```
Would be a good idea to prevent it from changing from the ID computer. A unique description would be cool as well.